conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -32,44 +32,59 @@
 import io.undertow.server.handlers.ResponseCodeHandler;
 import io.undertow.server.handlers.URLDecodingHandler;
 import io.undertow.util.Methods;
+import java.util.Collection;
 import java.util.List;
 import java.util.Set;
-import java.util.function.BiFunction;
 import java.util.stream.Collectors;
 
 /**
  * Conjure routing mechanism which can be registered as an Undertow {@link HttpHandler}.
  * This handler takes care of exception handling, tracing, and web-security headers
- * among {@link ConjureHandler#WRAPPERS other features}.
```

Comment:
nit: I'd just remove the link from this, it's not important to call out which features happen at what point in the lifecycle because that's an implementation detail.